server/base/src/main/java/org/apache/accumulo/server/master/balancer/TableLoadBalancer.java

```diff
@@ -47,8 +47,8 @@
 
   private TabletBalancer constructNewBalancerForTable(String clazzName, String table) throws Exception {
     String context = null;
-    if (null != configuration)
-      context = configuration.getTableConfiguration(table).get(Property.TABLE_CLASSPATH);
+    if (null != tabletBalancerEnvironment.getConfiguration())
```

Comment:
Hold that thought... it could be a problem with changes I made to the test... the test is supposed to override getTableConfiguration() to prevent it from calling out to ZK.